pkg/kube_events_manager/monitor.go

```diff
@@ -299,7 +288,6 @@ func (m *monitor) Start(parentCtx context.Context) {
 
 	if m.NamespaceInformer != nil {
 		m.NamespaceInformer.WithContext(m.ctx)
-		m.NamespaceInformer.WithSyncPeriod(m.informerSyncTime)
```

Comment:
NamespaceInformer is not using factory. Is it OK to delete this setter and use 0 as syncTime in underlying `wait.PollImmediateUntil`?

Reply:
Good question. I used `WithSyncPeriod` previously to tune this for tests to reduce the startup time for tests, Since we start this informer only once, tuning the sync period doesn't win us much time and look like a complication without benefits.

Reply:
It seems we can use DefaultSyncPeriod from factory in namespace_informer, see this commit: https://github.com/flant/shell-operator/commit/33f3aa9f45753b84f4d1586a6601d328ceaeeb4b
WDYT?

Reply:
Not necessarily, but ok. We can do it.